core/drivers/crypto/sub.mk

```diff
@@ -11,3 +11,5 @@ subdirs-$(CFG_STM32_CRYPTO_DRIVER) += stm32
 subdirs-$(CFG_ASPEED_CRYPTO_DRIVER) += aspeed
 
 subdirs-$(CFG_VERSAL_CRYPTO_DRIVER) += versal
+
+subdirs-$(CFG_HISILICON_PLAT_ACC) += hisilicon
```

Comment:
Would it be OK with you to name it `CFG_HISILICON_CRYPTO_DRIVER` for consistency with other platforms?